### DISCUSSION TITLE
How to add attributes to coord_system

### DISCUSSION THREAD

**Discussion by larsbarring** [SciTools/iris#5560] [Q&A] [answered]
Is there a way to add my own attributes to the `coord_system` of cube coordinates? 

Specifically, I would like to add `crs_wkt`(awaiting PR [#4719 ](https://github.com/SciTools/iris/pull/4719)) that is part  of CF since some time, and `proj_string` that might become part of CF sometime in the future (cf. https://github.com/cf-convention/discuss/issues/43).

**Comment by trexfeathers** [comment, marked as answer]
From @SciTools/peloton: `CoordSystem`s do not carry arbitrary attributes (nor can they be loaded/saved, of course). The Iris data model is pretty invested in the current approach, so making this idea work would need a big overhaul.

Xarray and NetCDF4 have the flexibility needed to do this, because they don't have any special handling for grid mapping variables.

Sorry for the inconvenience!
